### PR TITLE
v1.0 node release

### PR DESCRIPTION
**Pull request by ksyeo1010**

### CODE REVIEW

File: demo/nodejs/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+- Accurate; picoLLM Compression improves GPTQ by up to 98%.
+- Private; LLM inference runs 100% locally.
+- Cross-Platform
+- Runs on CPU and GPU
+- Free for open-weight models
+
+## Compatibility
+
+- Node.js 16+
+- Runs on Linux (x86_64), macOS (arm64, x86_64), Windows (x86_64), and Raspberry Pi (5, 4, and 3).
```
Comment (mrrostam) [inline demo/nodejs/README.md:19]:
```suggestion
- Runs on Linux (x86_64), macOS (arm64, x86_64), Windows (x86_64), and Raspberry Pi (5, 4).
```

File: binding/nodejs/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- Accurate; picoLLM Compression improves GPTQ by up to 98%.
+- Private; LLM inference runs 100% locally.
+- Cross-Platform
+- Runs on CPU and GPU
+- Free for open-weight models
+
+## Compatibility
+
+- Node.js 16+
+- Runs on Linux (x86_64), macOS (arm64, x86_64), Windows (x86_64), and Raspberry Pi (5, 4, and 3).
```
[the remark this answers is not in the document]
Comment (ksyeo1010) [reply binding/nodejs/README.md:19]: https://github.com/Picovoice/picollm/tree/main/binding/python#compatibility python has it

[the remark this answers is not in the document]
Comment (ksyeo1010) [reply binding/nodejs/README.md]: removed from there as well

[the remark this answers is not in the document]
Comment (mrrostam) [reply binding/nodejs/README.md]: it's a mistake, can you also fix it?

File: demo/python/README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ models. picoLLM Inference Engine is:
 ## Compatibility
 
 - Python 3.8+
-- Runs on Linux (x86_64), macOS (arm64, x86_64), Windows (x86_64), and Raspberry Pi (5, 4, and 3).
+- Runs on Linux (x86_64), macOS (arm64, x86_64), Windows (x86_64), and Raspberry Pi (5 and 3).
```
Comment (ErisMik) [inline demo/python/README.md:19]:
```suggestion
- Runs on Linux (x86_64), macOS (arm64, x86_64), Windows (x86_64), and Raspberry Pi (5 and 4).
```

**Pull request merged**